QEfficient/exporter/export_hf_to_cloud_ai_100.py

```diff
@@ -206,6 +206,10 @@ def convert_to_cloud_kvstyle(
 
     if tokenizer.pad_token_id is None:
         tokenizer.pad_token_id = tokenizer.eos_token_id
+        
+    # If Pad token is out of range of vocab size
+    if tokenizer.pad_token_id >= tokenizer.vocab_size:
```

Comment:
Is the `pad_token_id` being assigned on line 208? or it's coming from tokenizer?
If it's coming on line 208, we should move this in above `if` condition.

Reply:
makes sense

Reply:
So should we remove this from export_hf_to_cloud_ai_100 or keep it to support this as a LL API.